server/src/server.js

```diff
@@ -161,8 +165,8 @@ app.put("/data/:id/:domain", function (req, res) {
       return shot.update();
     }
     return null;
-  }).then(() => {
-    simpleResponse(res, "Saved", 200);
+  }).then((ok) => {
+    simpleResponse(res, JSON.stringify(shot.asJson()), 200);
```

Comment:
That is, I think I'd rather it just be a response like `{updateClipUrls: {clipId: {url: newUrl}}}` – since that's how it is being treated.